src/components/ec-file-dropzone/ec-file-dropzone.vue

```diff
@@ -1,59 +1,69 @@
 <template>
   <div
-    data-test="ec-file-dropzone"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-file-dropzone` : 'ec-file-dropzone'"
     class="ec-file-dropzone"
-    :class="{ 'ec-file-dropzone--dragging': isDragging }"
+    :class="{
+      'ec-file-dropzone--dragging': isDragging,
+      'ec-file-dropzone--is-disabled': isDisabled
+    }"
     @drop.prevent.stop="onComponentDrop"
   >
     <input
-      :id="id"
       ref="fileInput"
       data-test="ec-file-dropzone__input"
       type="file"
       class="ec-file-dropzone__input"
       multiple
+      :disabled="isDisabled"
       @change="onFileInputChange"
     >
 
     <svg
       height="40"
       width="50"
       class="ec-file-dropzone__img"
+      :class="{'ec-file-dropzone__img--is-disabled': isDisabled}"
     >
       <use href="../../assets/img/upload-cloud.svg#ec-upload-cloud" />
     </svg>
 
-    <h3 data-test="ec-file-dropzone__title">
+    <div
+      data-test="ec-file-dropzone__title"
+      class="ec-file-dropzone__title"
+      :class="{ 'ec-file-dropzone__title--is-disabled': isDisabled }"
+    >
       <slot name="title" />
-    </h3>
+    </div>
 
-    <label
```

Comment:
Why did we have to remove the label? Shouldn't each input have a label for accessibility reasons?

Reply:
Yes, is related to that, then we had a catch up with Milos and we said to make it a div or something.